farm/data_handler/processor.py

```diff
@@ -334,6 +338,9 @@ def _dict_to_samples(cls, dictionary: dict, all_dicts=None) -> [Sample]:
     def _sample_to_features(cls, sample: Sample) -> dict:
         raise NotImplementedError()
 
+    def _dict_to_samples_and_features(self, dictionary: dict, all_dicts=None) -> [Sample]:
```

Comment:
Why do you want to introduce a new method `_dict_to_samples_and_features`? 
Wouldn't it be cleaner to:
-  change `dict_to_samples` to also create the features 
- deprecate `_sample_to_features`
- eventually get rid of "tokenize_with_metadata"? 

In that case we would not have so much parallel code (once for slow, once for fast tokenizer) and it would be easier to maintain. Do you see any problems running the same code with the slow tokenizer?